reference/6/Microsoft.PowerShell.Core/Remove-Module.md

```diff
@@ -205,6 +205,23 @@ You can pipe module names and module objects to **Remove-Module**.
 This cmdlet does not generate any output.
 
 ## NOTES
+When removing a module, there is an event on the module that will execute.
+This event allows a module to react to being removed and perform some cleanup such as freeing up resources. Example:
+
+$OnRemoveScript = {
+
+  # perform cleanup
```

Comment:
Need to escape the `#` character. Markdown treats it as an H1 header. Change it to `\#`. We would have this problem in a code block. But you know the problem there.